src/components/Checkout/CheckoutReceipt.vue

```diff
@@ -125,7 +125,6 @@
 					v-if="kivaCards.length > 0"
 				>
 					For more details about all your Kiva Card purchases, please visit your
-					<!-- TODO: is this the tracking we want? -->
```

Comment:
I removed this because this comment node was causing mismatched rendering error in the browser (which causes a full render to run and makes weird things happen with hot reloading)